app/models/volunteer_op.rb

```diff
@@ -14,7 +14,6 @@ class VolunteerOp < ActiveRecord::Base
   scope :reachskills,          -> { where(source: 'reachskills') }
   scope :remote_only,          -> { where.not(source: 'local') }
 
```

Comment:
would it better if you revert the sort of method by name ? To have less files to review for the moment :)